VTEX - Buying Policies API.json

```diff
@@ -0,0 +1,2791 @@
+                    "variables": {
+                        "accountName": {
+                            "description": "Name of the VTEX account. Used as part of the URL.",
+                            "default": "apiexamples"
+                }
+                    }
+                }
+            ],
+            "post": {
+                "tags": [
+                    "Manual authorization"
+                ],
+                "summary": "Accept or deny rule",
+                "description": "Accepts or denies a pending dimension rule, a manual approval required for rules with `effectType: 2` (pending).\n\nWhen a rule requires manual authorization, you call this endpoint with a score value that is compared against the rule's `scoreInterval` thresholds:\n\n- If score ≥ `scoreInterval.accept`, the rule is accepted.\n- If score ≤ `scoreInterval.deny`, the rule is denied.\n- If the score falls between these values, the rule is bypassed.\n\nThe score is independent of the rule's expression evaluation. It's a value you provide to determine the authorization outcome.\n\n>ℹ️ The frontend/BFF fixes `scoreInterval` values as `accept: 10` and `deny: 5` for all rules, but these thresholds can be configured differently when creating dimensions via API. When using the standard frontend values, a score of `10` or higher will accept the rule.\r\n\r\nFor more information about approval dynamic, and dimension and rule priorities, see the [Buying Policies API overview](https://developers.vtex.com/docs/api-reference/buying-policies-api).\r\n\r\n## Permissions\n\nAny user or [API key](https://developers.vtex.com/docs/guides/authentication-overview#api-keys) must have at least one of the appropriate [License Manager resources](https://help.vtex.com/en/tutorial/license-manager-resources--3q6ztrC8YynQf6rdc6euk3) to be able to successfully run this request. Otherwise, they will receive a status code `403` error. These are the applicable resources for this endpoint:\n\n| **Product** | **Category** | **Resource** |\n| --------------- | ----------------- | ----------------- |\n| Buying Policies | OMS | **ManageBuyingPolicies** |\n\nThere are no applicable [predefined roles](https://help.vtex.com/en/tutorial/predefined-roles--jGDurZKJHvHJS13LnO7Dy) for this resource list. You must [create a custom role](https://help.vtex.com/en/tutorial/roles--7HKK5Uau2H6wxE1rH5oRbc#creating-a-role) and add at least one of the resources above in order to use this endpoint.\n\nTo learn more about machine authentication at VTEX, see [Authentication overview](https://developers.vtex.com/docs/guides/authentication-overview#machine-authentication).\n\n>❗ To prevent integrations from having excessive permissions, consider the [best practices for managing API keys](https://help.vtex.com/en/tutorial/best-practices-api-keys--7b6nD1VMHa49aI5brlOvJm) when assigning License Manager roles to integrations.",
```

Comment:
I think it's best to make it clear where the `scoreInterval.accept` and `scoreInterval.deny` values are defined.

Reply:
Also, are you sure that the permission for authorizing orders is `ManageBuyingPolicies`? It sounds weird. I think we should double check who is the persona sending this request (merchant or buyer org) and see if LM permissions or [storefront permissions](https://documenter.getpostman.com/view/2352637/2sAYkKGcHD#94f33c64-e8f2-49fc-91be-d42ad7c647c2) are applicable, or both.